hibernate-spatial/src/main/java/org/hibernate/spatial/SpatialPredicates.java

```diff
@@ -0,0 +1,492 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.spatial;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.Expression;
+import javax.persistence.criteria.Predicate;
+
+import org.hibernate.spatial.criterion.SpatialRestrictions;
+
+import org.locationtech.jts.geom.Geometry;
+
+/**
+ * JPA Criteria API support for hibernate-spatial.
+ *
+ * @author Daniel Shuy
+ */
+public class SpatialPredicates {
```

Comment:
It might be considerable to specify the class as `final` if it is meant only for static method container.

Reply:
Good point. I'm too spoiled by Lombok's `@UtilityClass` nowadays :P